rpc/src/main/java/org/dst/rpc/core/exception/DstException.java

```diff
@@ -0,0 +1,25 @@
+package org.dst.rpc.core.exception;
+
+
+public class DstException extends RuntimeException {
```

Comment:
`DstException ` has already existed.

Reply:
Ok, I will take an other name.

Reply:
I don't think there should be a description of DST. 
How about DRpcException？